dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/PoolTaskRestRepository.java

```diff
@@ -161,4 +132,57 @@ public void afterPropertiesSet() throws Exception {
                 new Link("/api/" + PoolTaskRest.CATEGORY + "/" + PoolTaskRest.NAME + "/search",
                         PoolTaskRest.NAME + "-search")));
     }
+
+    @PreAuthorize("hasAuthority('ADMIN')")
+    @SearchRestMethod(name = "findAllByItem")
+    public Page<PoolTaskRest> findAllByItem(@Parameter(value = "uuid", required = true) UUID itemUUID,
+           Pageable pageable) {
+        List<PoolTask> poolTasks = null;
+        try {
+            Context context = obtainContext();
+            Item item = itemService.find(context, itemUUID);
+            if (item == null) {
+                return null;
+            }
+            XmlWorkflowItem xmlWorkflowItem = xmlWorkflowItemService.findByItem(context, item);
+            if (xmlWorkflowItem == null) {
+                return null;
+            } else {
+                poolTasks = poolTaskService.find(context, xmlWorkflowItem);
+            }
+            return converter.toRestPage(poolTasks, pageable, utils.obtainProjection());
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+    }
+
+    @SearchRestMethod(name = "findByItem")
+    public PoolTaskRest findByItem(@Parameter(value = "uuid", required = true) UUID itemUUID) {
```

Comment:
Again, this should have `@PreAuthorize("hasAuthority('AUTHENTICATED')")`, and you should remove the check for a `currentUser` as it's no longer needed